src/main/java/com/ef/cim/objectmodel/Team.java

```diff
@@ -6,12 +6,14 @@
 import lombok.Setter;
 import lombok.ToString;
 
+import java.io.Serializable;
+
 @Getter
 @Setter
 @NoArgsConstructor
 @AllArgsConstructor
 @ToString
-public class Team {
```

Comment:
Why is this needed ?

Reply:
Same as the reason with KeycloakUser object.